ws_sub_orderfills.go

```diff
@@ -0,0 +1,27 @@
+package hyperliquid
+
+import "fmt"
+
+type OrderFillsSubscriptionParams struct {
+	User string
+}
+
+func (w *WebsocketClient) OrderFills(
+	params OrderFillsSubscriptionParams,
+	callback func([]WsOrderFill, error),
```

Comment:
`IsSnapshot` information shouldn't be forwarded to the client too?

Reply:
You're right, will update this.

Reply:
Updated this btw, @sonirico